src/main/java/uk/nhs/tis/trainee/notifications/model/ProgrammeMembership.java

```diff
@@ -39,6 +39,7 @@ public class ProgrammeMembership {
   private String programmeNumber;
   private String managingDeanery;
   private LocalDate startDate;
+  private LocalDate endDate;
```

Comment:
This is actually not used, since we rely on the PM's curricula's curriculumEndDate values.